.releaseindex.json

```diff
@@ -1,6 +1,8 @@
 {
-  "WindowsClassic":  "Libation\\.\\d+\\.\\d+\\.\\d+-win(dows)?-classic\\.zip",
-  "WindowsAvalonia": "Libation\\.\\d+\\.\\d+\\.\\d+-win(dows)?-chardonnay\\.zip",
-  "LinuxAvalonia":   "Libation\\.\\d+\\.\\d+\\.\\d+-linux-chardonnay\\.deb",
-  "MacOSAvalonia":   "Libation\\.app-macOS-x64-\\d+\\.\\d+\\.\\d+\\.tgz"
+  "WindowsClassic":      "Libation\\.\\d+\\.\\d+\\.\\d+-win(dows)?-classic\\.zip",
+  "WindowsAvalonia":     "Libation\\.\\d+\\.\\d+\\.\\d+-win(dows)?-chardonnay\\.zip",
+  "LinuxAvalonia":       "Libation\\.\\d+\\.\\d+\\.\\d+-linux-amd64\\.deb",

```

Comment:
I liked having the Chardonnay branding evident in the download file names. I'm often asked the difference between Classic and Chardonnay and it's easy to reply that Chardonnay is available on all platforms. Without it in the download name, I think this explanation is confusing.

Reply:
I added chardonnay to the built filenames.